jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -306,6 +309,23 @@ public void forEach(Consumer<? super StackFrame> action) {
 		 * @return StackTraceElement
 		 */
 		StackTraceElement toStackTraceElement();
+
+		/*[IF Java10]*/
+		/**
+		 * @return MethodType containing the parameter and return types for the associated method.
+		 */
+		default MethodType getMethodType() {
+			throw new UnsupportedOperationException();
+		}
+
+		/**
+		 * @return method descriptor string representing the type of this frame's method.
+		 */
+		default String getDescriptor() {
```

Comment:
The javadoc needs to be improved here as well

Reply:
Done.